ci/terraform/account-management/redis.tf

```diff
@@ -4,7 +4,7 @@ locals {
 
 resource "aws_elasticache_subnet_group" "account_management_redis_session_store" {
   name       = "${var.environment}-acct-mgmt-redis-session-store-cache-subnet"
-  subnet_ids = local.private_subnet_ids
+  subnet_ids = [local.private_subnet_ids, local.protected_subnet_ids]
```

Comment:
Why do we need to access the AM store?

Reply:
Ah, my bad, I think I saw "session_store" and thought this was the one it needed access to, before I changed the one in shared, and forgot to revert this